_docs/reporting/dora-metrics.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "DORA metrics"
+description: "Get insights for your deployments"
+group: reporting
+toc: true
+---
+
+DevOps is a collaboration paradigm and sometimes it is mistaken for being too
+abstract or too generic. In an effort to quantify the benefits of adopting DevOps,
+[Dora Research](https://www.devops-research.com/research.html#capabilities) (acquired by Google in 2018) has introduced 4 key metrics that
+define specific goals for improving the software lifecycle in companies that are
+interested in adopting DevOps.
+
+These metrics [are defined as](https://cloud.google.com/blog/products/devops-sre/using-the-four-keys-to-measure-your-devops-performance):
+
+* Deployment Frequency — How often an organization successfully releases to production
+* Lead Time for Changes — The amount of time it takes a commit to get into production
+* Change Failure Rate — The percentage of deployments causing a failure in production
+* Time to Restore Service — How long it takes an organization to recover from a failure in production
+
+Monitoring these metrics can help you identify delivery issues in your organization as they can detect bottlenecks among teams and show you where you need to optimize your workflows (either on a technical or organizational level).
+
```

Comment:
Why can't we have a title such "DORA in Codefresh" which will bring together everything else